### PR TITLE
49p scraper

### PR DESCRIPTION
**Pull request by billyfung**
scraper for 49 parallel roasters

### CODE REVIEW

File: scrapers/__init__.py [hunk 1]
```diff
@@ -3,3 +3,4 @@
 from .victrola import scrape_victrola
 from .heart import scrape_heart
 from .bluebottle import scrape_bluebottle
+from .forty9p import scrape_49p
```
Comment (jinpark) [inline scrapers/__init__.py]: whys the file name `fourty9p` and the function named `49p`

Comment (billyfung) [inline scrapers/__init__.py]: probably because i couldn't figure out how to call the file if it started with a number

Comment (jinpark) [inline scrapers/__init__.py]: then rename `scrape_49p` to `scrape_forty9p` or even better `scrape_fourtyninep`

Comment (jinpark) [inline scrapers/__init__.py]: Or their url is `49thcoffee` so how bout just `fortyninth`

**Comment by jinpark** [comment]
[LGTM](http://lgtm.in/i/a3vHiL93q)

**Pull request merged**